source/user/adc.rst

```diff
@@ -1,97 +1,180 @@
-.. _theory_adc:
+.. _user_adc:
 
 ************************************************
 Algebraic diagrammatic construction (ADC) scheme
 ************************************************
 
 *Modules*: :mod:`adc`
 
-Background: Propagator Theory
+Introduction
 =============================
+The algebraic diagrammatic construction theory (ADC(n)) is a post-Hartree-Fock
+method used for computing correlated excited states of molecules.
+:cite:`Schirmer1983,Schirmer1998`
+The ADC methods involve a perturbative expansion of a propagator followed
+by truncation at a certain order :math:`n` that defines the ADC(n) approximation.
+Depending upon the property being investigated, propagators can be of different
+types, which lead to different variants of the ADC method.
+At present, the `adc` module in PySCF can be used to calculate
+charged excitations, such as ionization potentials (IP-ADC) and
+electron affinities (EA-ADC) with different ADC(n) approximations
+(ADC(2), ADC(2)-X and ADC(3)). :cite:`Trofimov2005,Banerjee2019`
+The ADC(n) methods provide access to the
+excitation energies and their corresponding transition intensities via a
+'one-shot' calculation.
+
+A simple ADC(n) computation involves the calculation of the ground-state energy
+and wavefunction that correspond to those of the :math:`n`-th order
+Møller--Plesset perturbation theory (MPn), followed by the evaluation of the
+ADC(n) excitation energies (e.g., IP or EA) and the corresponding transition
+probabilities.
```

Comment:
Does this apply to all ADC methods?

Reply:
Yes, it does

Reply:
OK. So ADC becomes exact only in the case the MP series converges.